cln-rpc/src/model.rs

```diff
@@ -1928,7 +1982,6 @@ pub mod requests {
 	}
 	#[derive(Clone, Debug, Deserialize, Serialize)]
 	pub struct OfferRequest {
-	    pub amount: String,
```

Comment:
Why is the amount just removed here?

Reply:
Thanks for pointing it out. cln-rpc/src/model.rs file is auto generated. Previously, the amount was defined as "string" and now it is updated to "oneOf": [{"type": "msat_or_any"}, {"type": "currency"}]. Looks like model generation is unable to generate from `oneOf` type.

@cdecker Could you please help in generating the model.rs correctly? Or I can change it back to "string" type.

Reply:
Ignore my above message please, found the fix by adding Offer.amount in overrides array.